README.md

```diff
@@ -10,67 +10,73 @@ Django Major Event Log App [![Build Status](https://travis-ci.org/unt-libraries/
 About
 --------------------------
 
-This django appplication is designed to keep track of major premis events.
+This Django appplication is designed to keep track of major PREMIS events.
```

Comment:
Just noticed the typo on "appplication."  Also, does "keep track of major events in PREMIS format make the description more clear?